src/qt/rpcconsole.h

```diff
@@ -58,6 +58,7 @@ class RPCConsole: public QWidget
 #ifdef ENABLE_WALLET
     void addWallet(WalletModel* const walletModel);
     void removeWallet(WalletModel* const walletModel);
+    void SetCurrentWallet(WalletModel* const wallet_model);
```

Comment:
1. Should it be declared as a slot?

2. In the `qt` subdirectory we usually follow Qt's naming convention. Mind renaming please
```suggestion
    void setCurrentWallet(WalletModel* const wallet_model);
```
?

Reply:
The `const` has no effect here.